web-common/src/features/dashboards/workspace/DeployDashboardOverlay.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  import Dialog from "@rilldata/web-common/components/modal/dialog/Dialog.svelte";
+  import { projectShareStore } from "@rilldata/web-common/features/dashboards/dashboard-stores";
+
+  function close() {
+    projectShareStore.set(false);
+  }
+</script>
+
+<Dialog on:cancel={close} size="md">
+  <div class="" slot="body">
+    <div class="text-center text-2xl p-6">
+      Please use the CLI to deploy your project by running `rill deploy`
```

Comment:
Can we improve the copy and design of this part? Current dialog text looks a bit too big 

![image](https://user-images.githubusercontent.com/4402679/235106929-4cafaaec-98d9-4fc7-a327-0f33e785ed25.png)

Can we use the `CLICommandDisplay` component to show `rill deploy` with a copyable icon?

Also let's add a button which redirects the user to Rill Deploy docs.

Reply:
The design docs have these pretty big. We can check quickly with janet and figure this out.